openstack/messaging/v2/claims/requests.go

```diff
@@ -0,0 +1,73 @@
+package claims
+
+import (
+	"net/http"
+
+	"github.com/gophercloud/gophercloud"
+)
+
+// CreateOptsBuilder Builder.
+type CreateOptsBuilder interface {
+	ToClaimCreateRequest() (map[string]interface{}, string, error)
+}
+
+// CreateOpts params to be used with Create.
+type CreateOpts struct {
+	// The name of the queue to claim messages from.
+	QueueName string `json:"queue_name" required:"true"`
+
+	// Sets the TTL for the claim. When the claim expires un-deleted messages will be able to be claimed again.
+	TTL int `json:"ttl,omitempty"`
+
+	// Sets the Grace period for the claimed messages. The server extends the lifetime of claimed messages
+	// to be at least as long as the lifetime of the claim itself, plus the specified grace period.
+	Grace int `json:"grace,omitempty"`
```

Comment:
To make sure: 0 is not a valid Grace? If it is, this should be `*int`

Reply:
"The value of grace value must be between 60 and 43200 seconds (12 hours)."

So 0 is not a valid value.

Reply:
:+1: 

Thanks for confirming :)